docs/en/latest/plugins/csrf.md

```diff
@@ -0,0 +1,133 @@
+---
+title: csrf
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Summary
+
+- [**Description**](#description)
+- [**Attributes**](#attributes)
+- [**How To Enable**](#how-to-enable)
+- [**Test Plugin**](#test-plugin)
+- [**Disable Plugin**](#disable-plugin)
+
+## Description
+
+The `CSRF` plugin based on the `Double Submit Cookie` way, protect your API from CSRF attacks.
+
+## Attributes
+
+| Name             | Type    | Requirement | Default | Valid | Description                                                  |
+| ---------------- | ------- | ----------- | ------- | ----- | ------------------------------------------------------------ |
+|   name   |  string |    false    | `apisix-csrf-token`  |    | The name of the token in the generated cookie. |
```

Comment:
We don't use `true` or `false` for the requirement. Please see https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/gzip.md